docs/python/programming/math/vector.rst

```diff
@@ -549,6 +549,33 @@ Vector
     .. seealso::
         :meth:`Vector.record`, :meth:`Vector.play_remove`
     
+    Example of playing into an Iclamp for varying current:
+
+        .. code-block::
+            python
+        
+        from neuron import h
+        import pylab as plt, numpy as np
+        plt.ion()
+        h.load_file('stdrun.hoc')
+        sec = h.Section(name='sec')
+        sec.insert('pas')
```

Comment:
Personal preference, but I prefer:

    sec.insert(h.pas)

Reason being... you can query `h.pas` but the other is just a string.